package.json

```diff
@@ -10,13 +10,13 @@
     "install:citree": "cd tools/citree && npm install",
     "install:test-generator": "cd tools/test-generator && npm install",
     "build:config": "./scripts/build-config.mjs",
-    "build": "concurrently 'npm run build:templates' 'npm run build:ast' && npm run build:swc",
+    "build": "concurrently 'npm run build:templates' 'npm run build:ast' && npm run build:vite",
```

Comment:
Just a note here - I think I can get rid of this templates step by using [rollup-plugin-jscc](https://www.npmjs.com/package/rollup-plugin-jscc). This should make builds a lot smoother, get rid of the extra files, and give me a nicer DSL to boot. But that can happen in another PR.